src/equations/compressible_euler_1d.jl

```diff
@@ -9,8 +9,8 @@ end
 
 
 get_name(::CompressibleEulerEquations1D) = "CompressibleEulerEquations1D"
-varnames(::typeof(cons2cons), ::CompressibleEulerEquations1D) = @SVector ["rho", "rho_v1", "rho_e"]
-varnames(::typeof(cons2prim), ::CompressibleEulerEquations1D) = @SVector ["rho", "v1", "p"]
+varnames(::typeof(cons2cons), ::CompressibleEulerEquations1D) = ("rho", "rho_v1", "rho_e")
```

Comment:
Why not an `SVector` anymore? Is ther a downside to keeping it (or an upside to using plain tuples)?

Reply:
KISS: We don't need mathematical operations on the `varnames`, so I don't see a reason for using an `SVector` here.

Reply:
I'll KISS you right back! Sounds reasonable to me :+1: